examples/dial/example_dial.cpp

```diff
@@ -22,16 +22,15 @@
 #include <rpc/dial.hpp>
 
 using viam::robot::v1::Status;
-using Viam::SDK::Credentials;
-using Viam::SDK::Options;
+using namespace viam::cppsdk;
```

Comment:
I want to think a bit about whether `viam` is right at the top-level (I think so), and whether `cppsdk` is the right inner name (I suspect it is).